src/NServiceBus.Metrics/RawData/OccurrenceWriter.cs

```diff
@@ -0,0 +1,42 @@
+namespace NServiceBus.Metrics.RawData
+{
+    using System;
+    using System.IO;
+
+    static class OccurrenceWriter
+    {
+        const long Version = 1;
+
+        // WIRE FORMAT, Version: 1
+
+        //0                   1                   2                   3
+        //0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
```

Comment:
Do we need to tackle that problem now? For the metrics that we have for now we go from 1 message to 3 messages. I would not say it's a problem for now and I can see how can we optimize with versions 2 and later of the wire format.